### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by jnsdls**
## PR-Codex overview
This PR focuses on updating dependencies across various packages, including improvements in version numbers and handling of token styles in the `RenderCode` function.

### Detailed summary
- Updated dependency versions in `package.json` files for various packages.
- Increased size limit in `.size-limit.json` for `thirdweb`.
- Enhanced `RenderCode` function in `apps/portal/src/components/Document/Code.tsx` to handle undefined `htmlStyle` for `ThemedToken`.

> The following files were skipped due to too many changes: `pnpm-lock.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by jnsdls** [comment]
* **#5029** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5029?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jnsdls and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/thirdweb/package.json [hunk 1]
```diff
@@ -315,40 +279,40 @@
...
+    "@aws-sdk/client-lambda": "3.670.0",
     "@aws-sdk/credential-providers": "3.670.0",
-    "@biomejs/biome": "1.9.2",
+    "@biomejs/biome": "1.9.3",
     "@chromatic-com/storybook": "2.0.2",
     "@codspeed/vitest-plugin": "3.1.1",
     "@coinbase/wallet-mobile-sdk": "1.1.2",
     "@mobile-wallet-protocol/client": "0.0.3",
-    "@react-native-async-storage/async-storage": "1.24.0",
+    "@react-native-async-storage/async-storage": "2.0.0",
```
Comment (joaquim-verges) [inline packages/thirdweb/package.json:289]: need to double check this

Comment (jnsdls) [reply packages/thirdweb/package.json:289]: ah yeah lemme actually just downgrade this again

**Review by joaquim-verges — APPROVED** [review]

**Comment by jnsdls** [comment]
### Merge activity

* **Oct 14, 7:06 PM EDT**: A user  merged this pull request with the [Graphite merge queue](https://app.graphite.dev/merges?org=thirdweb-dev&repo=js).

**Pull request merged**